[testing] Unit tests failing on main

### Description

Seems we have main failing the unit test 

 The process cannot access the file 'C:\a\_work\1\s\.buildtasks\x64\libSkiaSharp.dll' 


### Link to public reproduction project repository

https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=11159245&view=results


### Relevant log output

```shell
"C:\a\_work\1\s\.buildtasks\x64/libSkiaSharp.dll". The process cannot access the file 'C:\a\_work\1\s\.buildtasks\x64\libSkiaSharp.dll' because it is being used by another process.
2025-03-10T04:46:53.4424761Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018: The "AdjustReferencedAssemblyVersion" task failed unexpectedly.
2025-03-10T04:46:53.4425847Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018: System.IO.IOException: The process cannot access the file 'C:\a\_work\1\s\.buildtasks\Svg.Skia.dll' because it is being used by another process.
2025-03-10T04:46:53.4426617Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
2025-03-10T04:46:53.4427402Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
2025-03-10T04:46:53.4428407Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
2025-03-10T04:46:53.4429313Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
2025-03-10T04:46:53.4430072Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
2025-03-10T04:46:53.4430957Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Mono.Cecil.ModuleDefinition.GetFileStream(String fileName, FileMode mode, FileAccess access, FileShare share)
2025-03-10T04:46:53.4431679Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Mono.Cecil.ModuleDefinition.Write(String fileName, WriterParameters parameters)
2025-03-10T04:46:53.4432346Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Mono.Cecil.AssemblyDefinition.Write(String fileName, WriterParameters parameters)
2025-03-10T04:46:53.4432984Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Mono.Cecil.AssemblyDefinition.Write(String fileName)
2025-03-10T04:46:53.4433609Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Mono.ApiTools.MSBuildTasks.AdjustReferencedAssemblyVersion.Execute()
2025-03-10T04:46:53.4434237Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Execute()
2025-03-10T04:46:53.4434998Z C:\a\_work\1\s\src\SingleProject\Resizetizer\src\Resizetizer.csproj(75,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(TaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)
```

--- a/src/SingleProject/Resizetizer/src/Resizetizer.csproj
+++ b/src/SingleProject/Resizetizer/src/Resizetizer.csproj
@@ -72,10 +72,15 @@
       <_CopyItems Include="@(_ResizetizerFiles)" />
     </ItemGroup>
     <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(_MauiBuildTasksLocation)%(RecursiveDir)%(_CopyItems.Arch)" ContinueOnError="true" Retries="0" />
+    <MakeDir Directories="$(IntermediateOutputPath)adjustments" />
     <AdjustReferencedAssemblyVersion
       Assembly="$(_AdjustmentsAssembly)"
       ReferencedAssembly="$(_AdjustmentsReferencedAssembly)"
-      OutputAssembly="$(_MauiBuildTasksLocation)Svg.Skia.dll" />
+      OutputAssembly="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" />
+    <Copy SourceFiles="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" DestinationFiles="$(_MauiBuildTasksLocation)Svg.Skia.dll" ContinueOnError="true" Retries="0" />
+    <ItemGroup>
+      <FileWrites Include="$(IntermediateOutputPath)adjustments\Svg.Skia.dll" />
+    </ItemGroup>
   </Target>
 
   <Import Project="$(MauiSrcDirectory)Workload\Shared\LibraryPacks.targets" />

